sdk/core/System.ClientModel/src/Message/ContentDispositionValue.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.ClientModel.Primitives;
+
+/// <summary>
+/// A class to represent the Content-Disposition value for a multipart/form-data request.
+/// </summary>
+public class ContentDispositionValue
```

Comment:
Curious: Did we consider and decide against the name `ContentDispositionHeaderValue` because of its length?

Reply:
No I didn't consider `ContentDispositionHeaderValue` but I think that's a much better name. I will update.